app/frontend/src/aquifers/layers.js

```diff
@@ -3,7 +3,8 @@ import L from 'leaflet'
 import ArtesianLegend from '../common/assets/images/artesian.png'
 import CadastralLegend from '../common/assets/images/cadastral.png'
 import EcocatWaterLegend from '../common/assets/images/ecocat-water.png'
-import GWaterLicenceLegend from '../common/assets/images/gwater-licence.png'
+import GWaterLicenceActiveLegend from '../common/assets/images/gwater-licence-active.svg'
+import GWaterLicenceInactiveLegend from '../common/assets/images/gwater-licence-inactive.svg'
 import OWellsInactiveLegend from '../common/assets/images/owells-inactive.png'
 import OWellsActiveLegend from '../common/assets/images/owells-active.png'
```

Comment:
It's usually more ideal to spell it out, unless this is how it is all across the gwells app?
GWaterLicence vs GroundWaterLicence
OWells vs ObservationWells (should it be ObservationWell since the others are not plural?)

Reply:
I was copying the structure of what was there. Personally it isn't a `class` so I wouldn't use an initial capital on the first letter either. There are a lot of short names used through-out GWELLS. Would you like to me to fix this as I go? Spelling things out and fixing weird capitalization?

Reply:
I think that would be beneficial to fix them as you go